### PR TITLE
[1LP][RFR] Revisited test_pull_splitter_persistence

### PR DESCRIPTION
**Pull request by mmojzis**
Added Utilization, InfraNetworking navigate_to.
Fixed uncollectif and blockers for test_pull_splitter_persistence
Added parametrized Bottlenecks, Utilization, InfraNetworking and deleted TemplatesImages.
Fixed Datastore navigate_to when there are no items in datastore and paginator is hidden.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 3d5870648d51ba65461924010d12846e7d51f206:

`cfme/tests/webui/test_splitter.py`:
- :red_circle: Line 4:1: **F401** *'cfme.control.explorer' imported but unused*
- :bangbang: Line 4:34: **E261** *at least two spaces before inline comment*
- :bangbang: Line 4:35: **E262** *inline comment should start with '# '*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 0057673fa92de111c893614b44d008ca967d0ecb:

`cfme/tests/webui/test_splitter.py`:
- :red_circle: Line 4:1: **F401** *'cfme.control.explorer' imported but unused*
- :bangbang: Line 4:34: **E261** *at least two spaces before inline comment*
- :bangbang: Line 4:35: **E262** *inline comment should start with '# '*
- :red_circle: Line 16:1: **F401** *'cfme.services.workloads' imported but unused*
- :red_circle: Line 17:1: **F401** *'cfme.optimize.utilization.Utilization' imported but unused*

Hmmm
:frowning: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/services/workloads.py [hunk 1]
```diff
@@ -62,6 +63,17 @@ def is_displayed(self):
                 "All VMs & Instances"])
 
 
+class WorkloadsDefaultView(WorkloadsView):
+    title = Text("#explorer_title_text")
+
+    @property
+    def is_displayed(self):
+        return (
+            super(WorkloadsVM, self).in_workloads and
```
Comment (mfalesni) [inline cfme/services/workloads.py]: `self.in_workloads` is sufficient

**Comment by mmojzis** [comment]
I was talking with @mfalesni and he told me that in Workloads is okay to navigate from Server, so I suppose this is alright. 
Maybe I can rewrite the other views in workloads to use my defaultworkload as prerequisite.

File: cfme/services/workloads.py (hunk 1 above)
Comment (psav) [inline cfme/services/workloads.py]: **REQUIRED:** let's get hsi changed and we can merge

**Comment by dajoRH** [comment]
Lint report for commit d80d573976ef7f0dc31e664f54eb25775fa4892c:

`cfme/tests/webui/test_splitter.py`:
- :red_circle: Line 17:1: **F401** *'cfme.optimize.utilization.Utilization' imported but unused*
- :bangbang: Line 34:14: **E201** *whitespace after '['*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by mmojzis** [comment]
@psav Your requested change is done.

**Comment by psav** [comment]
Just waiting on PRT

**Comment by mmojzis** [comment]
@psav Some views were changed again, so I had to add another "Default" views to make splitters working. Not it passed through prt so it should be mergeable.

**Pull request merged**